src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerMessageTest.cs

```diff
@@ -193,6 +193,34 @@ public void LogMessages(Delegate messageDelegate, int argumentCount)
             Assert.Equal(expectedToString, actualLogValues.ToString());
         }
 
+        [Fact]
+        public void LogMessage_WithNullParameter_DoesNotMutateArgument()
+        {
+            // Arrange
+            var format = "TestMessage {param1} {param2} {param3}";
+            var param1 = "foo";
+            var param2 = (string)null;
+            int param3 = 10;
+            var testSink = new TestSink();
+            var testLogger = new TestLogger("testlogger", testSink, enabled: true);
+
+            // Act
+            testLogger.LogInformation(format, param1, param2, param3);
+
+            // Assert
+            Assert.Single(testSink.Writes);
+            var write = testSink.Writes.First();
```

Comment:
>var [](start = 12, length = 3)

please specify the type instead of var in the lines that is not clear what type the RHS will return

Reply:
I was a bit confused on if I should use var or not as I've read the coding guidelines for the dotnet repos that states that the type should be specified, but also to keep the existing style of a file, even if it breaks the guidelines. 

In this case I copied the structure of the test from another test to keep the style consistent with the others, so that's why I kept using var here. I'll fix my added test to use explicit types, but leave the existing ones in the file as they are then?